src/utils/deprecated/deprecated.test.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { render } from 'enzyme';
+import { deprecated } from './deprecated';
+
+describe('deprecated', () => {
+  const warning = 'This component is deprecated in favor of another.';
+
+  afterAll(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should render component with deprecation warning', () => {
+    console.warn = jest.fn();
+
+    const LogoComponent = () => <div />;
+    const Logo = deprecated(warning)(LogoComponent);
+
+    const component = render(<Logo />);
+    expect(component).toMatchSnapshot();
+    expect(console.warn).toHaveBeenCalledWith(`[DEPRECATED] ${warning}`);
```

Comment:
Should this be using `getDeprecatedMessage`?

Reply:
Yeap, I corrected it, thanks!